lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -66,6 +66,17 @@
   .form-actions {
     padding-top: gutter();
   }
+
+  &.-signup {
+    width: 30%;
+    min-width: 256px;
+    margin: 1.5%;
+    display: inline-block;
+
+    .beta-card__row:first-of-type {
+      margin-top: 0;
+    }
+  }
```

Comment:
I'm confused... what does this have to do with the row? It's dealing w/ just the card...